packages/synapse-react-client/src/components/UserCard/UserCardMedium.tsx

```diff
@@ -101,9 +102,9 @@ export function UserCardMedium({
   let name = ''
   const linkLocation = link
     ? link
-    : `${PRODUCTION_ENDPOINT_CONFIG.PORTAL}Profile:${userProfile.ownerId}`
+    : `${PRODUCTION_ENDPOINT_CONFIG.PORTAL}Profile:${ownerId}`
   // linkLocation is overriden by custom click handler
-  const email = `${userName}@synapse.org`
+  const email = userName ? `${userName}@synapse.org` : undefined
```

Comment:
Could we update the type definition, so we can catch all of the other places where we may have a similar error?

https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/fdcfcc8909d7f5625e55bc0997700b7e168c9a6e/packages/synapse-types/src/UserProfile.ts#L27

Reply:
Absolutely!  I thought I made this change, but I accidentally made the change to the output UserProfile.d.ts file instead